.github/workflows/daily-digest.yml

```
@@ -0,0 +1,278 @@
+              labels: 'blocked',
+              state: 'open',
+              per_page: 100
+            });
+            
+            const { data: reviewPRs } = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              per_page: 100
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)